quarkus-extension/pom.xml

```diff
@@ -55,5 +55,21 @@
         <module>engine</module>
       </modules>
     </profile>
+
+    <profile>
+      <id>distro</id>
+      <build>
+        <pluginManagement>
+          <plugins>
+            <plugin>
+              <artifactId>maven-surefire-plugin</artifactId>
+              <configuration>
+                <skipTests>true</skipTests>
```

Comment:
❓ With this implementation the tests will be skipped for the JDK 17 stages, are we ok with that?

Reply:
The new Quarkus UNIT stage will run the tests using JDK 17 already 👍

Reply:
Ok, so JDK 11 will be skipped.

Reply:
We're fine with this, see [Slack message](https://camunda.slack.com/archives/C05T6BJ8RDW/p1695284650105929?thread_ts=1695036904.148599&cid=C05T6BJ8RDW) and following.

Reply:
👍